### ISSUE TITLE
openjdk8_j9_special.system_x86-64_windows Cannot delete workspace

### ISSUE THREAD

**Issue by pshipton** [eclipse/openj9#6599]
https://ci.eclipse.org/openj9/job/Test_openjdk8_j9_special.system_x86-64_windows_Nightly/1
```
05:40:06  + printenv
05:40:06  TIME_LIMIT=10
05:40:06  USERDOMAIN=OJ9-WIN2012R2-3
05:40:06  OS=Windows_NT
05:40:06  COMMONPROGRAMFILES=C:\Program Files\Common Files
05:40:06  KNOWN_HOSTS=github.com github.ibm.com
05:40:06  PROCESSOR_LEVEL=6
05:40:06  PSModulePath=%ProgramFiles%\WindowsPowerShell\Modules;C:\Windows\system32\WindowsPowerShell\v1.0\Modules
05:40:06  RUN_DISPLAY_URL=https://ci.eclipse.org/openj9/job/Test_openjdk8_j9_special.system_x86-64_windows_Nightly/1/display/redirect
05:40:06  JENKINS_URL=https://ci.eclipse.org/openj9/
05:40:06  CommonProgramW6432=C:\Program Files\Common Files
05:40:06  EXECUTOR_NUMBER=0
05:40:06  SSH_CONNECTION=198.41.30.203 52133 169.55.170.74 22
05:40:06  INCLUDE=C:\openjdk\windbgtools\sdk\inc
05:40:06  CommonProgramFiles(x86)=C:\Program Files (x86)\Common Files
05:40:06  _=/usr/bin/printenv
05:40:06  FP_NO_HOST_CHECK=NO
05:40:06  BUILD_ID=1
05:40:06  BUILD_IDENTIFIER=Nightly
05:40:06  PUBLIC=C:\Users\Public
05:40:06  RUN_CHANGES_DISPLAY_URL=https://ci.eclipse.org/openj9/job/Test_openjdk8_j9_special.system_x86-64_windows_Nightly/1/display/redirect?page=changes
05:40:06  PERSONAL_BUILD=false
05:40:06  IS_PARALLEL=true
05:40:06  JENKINS_NODE_COOKIE=827801fc-2290-4e0a-b954-a0114a35f3be
05:40:06  JRE_IMAGE=C:\Users\jenkins\workspace\Test_openjdk8_j9_special.system_x86-64_windows_Nightly/openjdkbinary/j2re-image
05:40:06  USERNAME=jenkins
05:40:06  JAVA_HOME=C:\Program Files\Java\jdk8u202-b08
05:40:06  LOGONSERVER=\\OJ9-WIN2012R2-3
05:40:06  JDK_IMPL=openj9
05:40:06  PROCESSOR_ARCHITECTURE=AMD64
05:40:06  JOB_BASE_NAME=Test_openjdk8_j9_special.system_x86-64_windows_Nightly
05:40:06  OPENJ9_BRANCH=master
05:40:06  LOCALAPPDATA=C:\Users\jenkins\AppData\Local
05:40:06  ARTIFACTORY_SERVER=ci-eclipse-openj9
05:40:06  HUDSON_COOKIE=a51d9f9d-6c06-4677-8e50-aed98aaefa74
05:40:06  COMPUTERNAME=OJ9-WIN2012R2-3
05:40:06  USER=jenkins
05:40:06  ADOPTOPENJDK_BRANCH=master
05:40:06  SYSTEMDRIVE=C:
05:40:06  USERPROFILE=C:\Users\jenkins
05:40:06  PATHEXT=.COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC
05:40:06  SYSTEMROOT=C:\Windows
05:40:06  BUILD_NUMBER=1
05:40:06  PROCESSOR_IDENTIFIER=Intel64 Family 6 Model 63 Stepping 2, GenuineIntel
05:40:06  WORKSPACE=C:\Users\jenkins\workspace\Test_openjdk8_j9_special.system_x86-64_windows_Nightly
05:40:06  KEEP_REPORTDIR=false
05:40:06  PWD=/cygdrive/c/Users/jenkins/workspace/Test_openjdk8_j9_special.system_x86-64_windows_Nightly
05:40:06  HUDSON_URL=https://ci.eclipse.org/openj9/
05:40:06  HOME=/home/jenkins
05:40:06  NODE_NAME=win2012r2-x86-3
05:40:06  PERF_ROOT=C:\Users\jenkins\workspace\Test_openjdk8_j9_special.system_x86-64_windows_Nightly/../../benchmarks
05:40:06  HUDSON_SERVER_COOKIE=e4ce3607d4755b7e
05:40:06  SSH_CLIENT=198.41.30.203 52133 22
05:40:06  JENKINS_HOME=/home/hudson/genie.openj9/.jenkins
05:40:06  JOB_NAME=Test_openjdk8_j9_special.system_x86-64_windows_Nightly
05:40:06  BUILD_LIST=systemtest
05:40:06  VS100COMNTOOLS=c:\Program Files (x86)\Microsoft Visual Studio 10.0\Common7\Tools\
05:40:06  KEEP_WORKSPACE=false
05:40:06  ALL_NODES=proxy_worker aix71-p8-2 aix71-p8-3 aix71-p8-4 ub16-390-1 ub16-390-2 ub16-390-3 ub16-390-4 ub16-ppcle-1 ub16-ppcle-2 ub16-ppcle-3 win2012r2-x86-1 win2012r2-x86-2 win2012r2-x86-3 win2012r2-x86-4 win2012r2-x86-5 win2012r2-x86-6 ub16x64j91 ub16x64j92 ub16x64j93 ub16x64j94 ub16x64j95 ub16x64j96 ub16x64j97 ub16x64j98 ub16x64j99 ub16x64j910 ub16p8j91 ub16p8j92 ub16p8j93 ub16p8j94 ub16p8j95 ub16p8j96 osx1011-x86-1 osx1011-x86-2 osx1013-x86-3 osx1013-x86-4
05:40:06  VS120COMNTOOLS=C:\Program Files (x86)\Microsoft Visual Studio 12.0\Common7\Tools\
05:40:06  PROCESSOR_REVISION=3f02
05:40:06  HUDSON_HOME=/home/hudson/genie.openj9/.jenkins
05:40:06  JOB_DISPLAY_URL=https://ci.eclipse.org/openj9/job/Test_openjdk8_j9_special.system_x86-64_windows_Nightly/display/redirect
05:40:06  ARTIFACTORY_REPO=ci-eclipse-openj9
05:40:06  SDK_RESOURCE=upstream
05:40:06  LIB=C:\openjdk\windbgtools\sdk\lib
05:40:06  SPEC=win_x86-64_cmprssptrs
05:40:06  ITERATIONS=1
05:40:06  NUMBER_OF_PROCESSORS=8
05:40:06  TEST_JDK_HOME=C:\Users\jenkins\workspace\Test_openjdk8_j9_special.system_x86-64_windows_Nightly/openjdkbinary/j2sdk-image
05:40:06  ProgramW6432=C:\Program Files
05:40:06  OPENJ9_SHA=ba3a102a224f44f33dcd634fdd86803678dc2632
05:40:06  MAIL=/var/spool/mail//jenkins
05:40:06  CUSTOMIZED_SDK_URL_CREDENTIAL_ID=ab89294b-5ba1-48e9-8c85-107daca5a2e9
05:40:06  AUTO_DETECT=true
05:40:06  COMSPEC=C:\Windows\system32\cmd.exe
05:40:06  APPDATA=C:\Users\jenkins\AppData\Roaming
05:40:06  BUILD_URL=https://ci.eclipse.org/openj9/job/Test_openjdk8_j9_special.system_x86-64_windows_Nightly/1/
05:40:06  SHELL=/bin/bash
05:40:06  TERM=cygwin
05:40:06  STAGE_NAME=setupParallelEnv
05:40:06  OPENJ9_REPO=https://github.com/eclipse/openj9.git
05:40:06  JOB_URL=https://ci.eclipse.org/openj9/job/Test_openjdk8_j9_special.system_x86-64_windows_Nightly/
05:40:06  ARTIFACTORY_ROOT_DIR=Test
05:40:06  CUSTOMIZED_SDK_URL=https://140-211-168-230-openstack.osuosl.org/artifactory/ci-eclipse-openj9/Build_JDK8_x86-64_windows_Nightly/107/OpenJ9-JDK8-x86-64_windows-20190727-024854.tar.gz https://140-211-168-230-openstack.osuosl.org/artifactory/ci-eclipse-openj9/Build_JDK8_x86-64_windows_Nightly/107/native-test-libs.tar.gz
05:40:06  JDK_VERSION=8
05:40:06  BUILD_DISPLAY_NAME=#1
05:40:06  ADOPTOPENJDK_REPO=https://github.com/AdoptOpenJDK/openjdk-tests.git
05:40:06  WINDIR=C:\Windows
05:40:06  ProgramData=C:\ProgramData
05:40:06  SHLVL=3
05:40:06  PROGRAMFILES=C:\Program Files
05:40:06  BUILD_TAG=jenkins-Test_openjdk8_j9_special.system_x86-64_windows_Nightly-1
05:40:06  ALLUSERSPROFILE=C:\ProgramData
05:40:06  NODE_LABELS=ci.geo.softlayer ci.role.build ci.role.test hw.arch.x86 sw.os.windows sw.os.windows.2012 win2012r2-x86-3
05:40:06  LOGNAME=jenkins
05:40:06  TARGET=special.system
05:40:06  ANT_HOME=C:\apache-ant\apache-ant-1.10.3
05:40:06  ProgramFiles(x86)=C:\Program Files (x86)
05:40:06  PATH=/usr/bin:/usr/bin:/cygdrive/c/Windows/system32:/cygdrive/c/Windows:/cygdrive/c/Windows/System32/Wbem:/cygdrive/c/Windows/System32/WindowsPowerShell/v1.0:/cygdrive/c/Program Files/Microsoft Windows Performance Toolkit:/cygdrive/c/apache-ant/apache-ant-1.10.3/bin:/cygdrive/c/Program Files/Java/jdk8u202-b08/bin:/cygdrive/c/Strawberry/perl/bin:/cygdrive/c/Program Files/Microsoft SQL Server/110/Tools/Binn:/cygdrive/c/Program Files (x86)/Microsoft SDKs/TypeScript/1.0:/cygdrive/c/Strawberry/perl/bin:/cygdrive/c/Program Files/Java/jdk8u202-b08/bin:/cygdrive/c/apache-ant/apache-ant-1.10.3/bin:/cygdrive/c/OpenSSL-1.1.1-x86_32/bin:/cygdrive/c/OpenSSL-1.1.1-x86_64/bin
05:40:06  HOMEDRIVE=C:
05:40:06  HOMEPATH=\cygwin64\home\jenkins
05:40:06  JENKINS_SERVER_COOKIE=durable-5dbe8b2e68d5f6645112b231ea5e468b
05:40:06  VS110COMNTOOLS=C:\Program Files (x86)\Microsoft Visual Studio 11.0\Common7\Tools\
[Pipeline] dir
05:40:06  Running in C:\Users\jenkins\workspace\Test_openjdk8_j9_special.system_x86-64_windows_Nightly/openjdk-tests/systemtest
[Pipeline] {
[Pipeline] sh
05:40:07  + ls -d daaLoadTest/ jlm/ lambdaLoadTest/ mathLoadTest/ mauveLoadTest/ modularity/ otherLoadTest/ sharedClasses/
[Pipeline] }
[Pipeline] // dir
[Pipeline] echo
05:40:07  testSubDirSize is 8, testSubDirs is [daaLoadTest/, jlm/, lambdaLoadTest/, mathLoadTest/, mauveLoadTest/, modularity/, otherLoadTest/, sharedClasses/], running test in parallel mode
[Pipeline] }
[Pipeline] // timestamps
[Pipeline] }
[Pipeline] // stage
[Pipeline] cleanWs
[WS-CLEANUP] Deleting project workspace...
[WS-CLEANUP] Deferred wipeout is used...
ERROR: Cannot delete workspace :Unable to delete 'C:\Users\jenkins\workspace\Test_openjdk8_j9_special.system_x86-64_windows_Nightly'. Tried 3 times (of a maximum of 3) waiting 0.1 sec between attempts.
[Pipeline] }
[Pipeline] // timeout
[Pipeline] }
[Pipeline] // node
[Pipeline] }
[Pipeline] // stage
[Pipeline] End of Pipeline
ERROR: Cannot delete workspace: Unable to delete 'C:\Users\jenkins\workspace\Test_openjdk8_j9_special.system_x86-64_windows_Nightly'. Tried 3 times (of a maximum of 3) waiting 0.1 sec between attempts.
Finished: FAILURE
```

**Comment by pshipton** [comment]
@AdamBrousseau @llxia

**Comment by pshipton** [comment]
Restarted but same problem
https://ci.eclipse.org/openj9/job/Test_openjdk8_j9_special.system_x86-64_windows_Nightly/2/

**Comment by AdamBrousseau** [comment]
I don't think this is a machine issue @llxia .

**Comment by llxia** [comment]
@LongyuZhang could you investigate this? This only happens on windows. I suspect the subdir is not released, so the workspace cannot be deleted on wins. 
https://github.com/AdoptOpenJDK/openjdk-tests/blob/d95bab62f5c542bfc182d20db5bf99738f2c419f/buildenv/jenkins/JenkinsfileBase#L116

**Comment by llxia** [comment]
Thanks for Longyu's help.

We noticed that `special.system` runs fine on windows internally `Test-special.system-JDK8-win_x86-64_cmprssptrs`.

@AdamBrousseau do the windows machines configured differently?

**Comment by llxia** [comment]
Talked with @jdekonin and @vsebe , we are going to try `cleanWs notFailBuild: true, disableDeferredWipeout: true, deleteDirs: true` instead of `cleanWs()`

**Comment by llxia** [comment]
Above cmd worked https://ci.eclipse.org/openj9/job/Test_openjdk8_j9_special.system_x86-64_windows_Nightly/12/console
